app/models/work.rb

```diff
@@ -188,9 +207,15 @@ def valid_to_submit
     errors.clear
     validate_ark
     validate_metadata
-    if deposit_uploads.length > 20
-      errors.add(:base, "Only 20 files may be uploaded by a user to a given Work. #{deposit_uploads.length} files were uploaded for the Work: #{ark}")
+
+    if pre_curation_uploads.length > 20
```

Comment:
Do we need to check the total of post and pre curated so we do not allow them to add more than 20 and then can not transfer them when they get approved?